lms/templates/registration/password_reset_confirm.html

```diff
@@ -3,148 +3,70 @@
-                We're sorry, but this version of your browser is not supported. Try again using a different browser or a newer version of your browser.
-            {% endblocktrans %}
-          </h3>
-        </div>
-
-        {% if err_msg %}
-        <div role="alert" class="status message submission-error" style="display: block;">
-        {% else %}
-        <div role="alert" class="status message submission-error" style="display: none;">
-        {% endif %}
-          <h3 class="message-title">{% trans "The following errors occurred while processing your registration: " %}</h3>
-          <ul class="message-copy">
-            {% if err_msg %}
-              <li>{{err_msg}}</li>
+<div id="password-reset-confirm-container" class="login-register">
+    <section id="password-reset-confirm-anchor" class="form-type">
+        <div id="password-reset-confirm-form" class="form-wrapper">
+            <div class="status submission-error {% if not err_msg %} hidden {% endif %}" aria-live="polite">
+                <h4 class="message-title">{% trans "Error Resetting Password" %}</h4>
+                <ul class="message-copy">
+                {% if err_msg %}
+                    <li>{{err_msg}}</li>
+                {% else %}
+                    <li>{% trans "You must enter and confirm your new password." %}</li>
+                    <li>{% trans "The text in both password fields must match." %}</li>
+                {% endif %}
+                </ul>
+            </div>
+            {% if validlink %}
+
+            <form role="form" id="passwordreset-form" method="post" action="">{% csrf_token %}
+                <div class="section-title lines">
+                    <h2>
+                        <span class="text">
+                        {% trans "Reset Your Password" %}
+                        </span>
+                    </h2>
+                </div>
+
+                <p class="action-label">
+                    {% trans "Enter and confirm your new password." %}
+                </p>
+
+                <div class="form-field new_password1-new_password1">
+                    <label for="new_password1">{% trans "New Password" %}</label>
+                    <input id="new_password1" type="password" name="new_password1" placeholder="*****" />
+                </div>
+                <div class="form-field new_password2-new_password2">
+                    <label for="new_password2">{% trans "Confirm Password" %}</label>
+                    <input id="new_password2" type="password" name="new_password2" placeholder="*****" />
+                </div>
+
+                <button type="submit" class="action action-primary action-update js-reset">{% trans "Reset My Password" %}</button>
+            </form>
             {% else %}
-              <li>{% trans "You must complete all fields." %}</li>
-              <li>{% trans "The two password fields didn't match." %}</li>
+            <div class="status submission-error" aria-live="polite">
```

Comment:
because this ```div``` has interactive content (the links to reset password or sign in), it should not be an aria-live region.   In fact, it most likely gets injected at initial page load, so it most likely wouldn;'t work as a live region.  I would remove the aria-live attribute above.